server/interpreter.js

```diff
@@ -2734,6 +2859,92 @@ Interpreter.prototype.Server.prototype.unlisten = function(onClose) {
   throw Error('Inner class method not callable on prototype');
 };
 
+/**
+ * @constructor
```

Comment:
A high-level comment here about Box would be good.

Reply:
As with `Object`, `Function`, etc., the high-level comment is with the actual implementation, at `intrp.Box = ...` in `Interpreter.prototype.installTypes`.

This isn't strictly correct according to the styleguide, but it would have me put the main documentation here and then the implementation would end up with neither the JSDoc comment nor even the type declarations (because the constructor signature is identical).